src/com/palmergames/bukkit/towny/command/TownyAdminCommand.java

```diff
@@ -1425,6 +1433,89 @@ else if (split[1].equalsIgnoreCase("rank")) {
 						sender.sendMessage(ChatTools.formatCommand(Translation.of("admin_sing"), "/townyadmin nation rank remove [resident] [rank] ", ""));
 						return;
 				}
+			} else if (split[1].equalsIgnoreCase("ally")) {
+				if (split.length < 4)
+					return;
+				
+				Nation ally = townyUniverse.getNation(split[3]);
+				if (ally == null) {
+					TownyMessaging.sendErrorMsg(player, Translation.of("msg_err_invalid_name", split[3]));
+					return;
+				}
+
+				if (split[2].equalsIgnoreCase("add")) {
+					if (!nation.hasAlly(ally)) {
+						if (nation.hasEnemy(ally))
+							nation.removeEnemy(ally);
+						
+						if (ally.hasEnemy(nation))
+							ally.removeEnemy(nation);
+						
+						nation.addAlly(ally);
+						nation.save();
+
+						ally.addAlly(nation);
+						ally.save();
+
+						plugin.resetCache();
+						TownyMessaging.sendPrefixedNationMessage(nation, Translation.of("msg_added_ally", ally.getName()));
+						TownyMessaging.sendPrefixedNationMessage(ally, Translation.of("msg_added_ally", nation.getName()));
+						TownyMessaging.sendMessage(player, Translation.of("default_towny_prefix") + Translation.of("msg_ta_allies_enemies_updated", nation.getName()));

```

Comment:
Throughout TownyAdminCommand where you have message feedback going to player you should be using getSender() which will send it to the player or console, dependent on where the command originated from.

Reply:
And instead of using sendMessage() and including the prefix you can use sendMsg() which should prefix things automatically.

Reply:
Fixed, already thought it was weird that the default prefix wasn't showing up.